### PR TITLE
Update and fix Android Linux makefile sample for template project

### PR DESCRIPTION
**Pull request by itarato**
## Tl;dr

This PR attempts to update the Android-Linux makefile on the template project so it runs with the latest state of tooling and version.

## Changes

- added Android NDK reference in order to use their binaries at a few points.

It seems some linked bins and sources were using local or other folders that either do not contain those binaries or moved in recent updates.

- corrected a few architecture related binary file references to the correct files
- added CC/AR definitions to x86 / x86_86
- swapped the order of zipaligning and signing

With the original order I was getting a lot of aligning errors and digging on error reports it seems zipaligning has to be done on the unsigned apk just before signing

- changed signing from jarsigner to apksigner as the former one is not supported anymore
- minor shell command syntax issues

## Test

I've tested it on latest Android version both emulator and real device, all works.

### CODE REVIEW

**Comment by raysan5** [comment]
@itarato Nice! Thank you very much for the review and the detailed explanation! 👍😄

**Comment by itarato** [comment]
Pleasure :)
